actions/views/channel.js

```diff
@@ -15,6 +15,7 @@ import {getLastViewedChannelName} from 'selectors/local_storage';
 import {browserHistory} from 'utils/browser_history';
 import {ActionTypes} from 'utils/constants.jsx';
 import {isMobile} from 'utils/utils.jsx';
+import Constants from '../../utils/constants';
```

Comment:
isn't this a dup import from two lines above? not that is blocker but you could use
```js
import {General} from 'mattermost-redux/constants'

channel.type === General.DM_CHANNEL
```

Reply:
Ahh that's my auto-importer doing that. Let me fix it